repos/fdbt-site/src/pages/home.tsx

```diff
@@ -82,6 +83,23 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): {} => {
         redirectTo(ctx.res, '/multipleOperators');
     }
 
+    if (sessionNoc) {
```

Comment:
If the user has multiple Nocs, they will be redirected to multipleOperators and this session attribute wont be set. That means we need to add this logic to the /multipleOperators API (when the user selects a NOC to use).

Reply:
as discussed, not needed anymore